### DISCUSSION TITLE
Not supported by this device / controller-specific parameter list

### DISCUSSION THREAD

**Discussion by marcelpettigrew** [fredlcore/BSB-LAN#535] [Q&A] [answered]
Hi,

I have a problem with the controller-specific parameter list. 
The file is created directly and has the following content:
`
Version: 3.0.4-20221222224441
Scan for devices...
Device address found: 0
Test device address 0...

Complete dump:
DC C2 00 0B 06 3D 05 00 01 62 EB
DC C2 00 0B 06 3D 05 00 64 5E E8

Not supported by this device. No problem.

Fertig.
`
Only very few parameters are displayed in the web interface, but a connection is present, as I get the temperature sensor displayed, for example. 

![image](https://user-images.githubusercontent.com/122126371/211101957-3c3c3c76-5624-44ec-a0e5-90140ab0529f.png)

![image](https://user-images.githubusercontent.com/122126371/211102086-84f70904-12b1-4117-a43b-9c915ca7bbd0.png)

![image](https://user-images.githubusercontent.com/122126371/211102209-6a2e9131-2400-427b-881a-a3b1449317f4.png)

Setup:
- Brötje BSW8E (RVS61) -> BSB
- Olimex ESP32 EVB
- BSB-LAN, Version 3.0.4-20221222224441

Can someone help me how I can get the list to be read out?

**Comment by fredlcore** [comment]
Hm, something seems wrong, because the routine finds a device (id 0) which means the connection is ok, but when querying it, it doesn't get a response from the heater. That is very odd because even older heaters which do not support the device specific parameter list at least respond with their ID, but yours does none of that.
What happens if you query parameter 6224, 6225 and 6226? If you get responses to these, the dump has to show at least a little bit more than that above. If it still does not, then I'm at loss here. My only suggestion would be to flash and connect everything again from scratch.

**Comment by marcelpettigrew** [comment]
That seems to cap
![image](https://user-images.githubusercontent.com/122126371/211156259-19ddb993-d1ef-4941-8768-25bd94685be2.png)

I'll try to reflash the system.

**Comment by fredlcore** [comment]
And if right after that query you try to request the "Reglerspezifische Parameterliste"?

**Comment by marcelpettigrew** [comment]
unfortunately the same result in the log

**Comment by marcelpettigrew** [comment]
![image](https://user-images.githubusercontent.com/122126371/211157012-617b3607-c9cc-4c34-9045-41981d754f97.png)

with other parameters there is no feedback. Like here e.g. 712

**Comment by fredlcore** [comment]
Can you attach the new file here, please, along with s screenshot of the configuration page and the configuration file? 

712 is empty in the default version.

**Comment by marcelpettigrew** [comment]
[BSB-LAN-232-000-0.txt](https://github.com/fredlcore/BSB-LAN/files/10368195/BSB-LAN-232-000-0.txt)

![image](https://user-images.githubusercontent.com/122126371/211192463-85f33f2b-ab12-4c6e-a308-d3378897215a.png)
![image](https://user-images.githubusercontent.com/122126371/211192481-343d0e65-46c3-4280-bec8-37e18db62e11.png)
![image](https://user-images.githubusercontent.com/122126371/211192499-865d80d3-de7a-4fb6-9aec-ddb5546d6df4.png)
[BSB_LAN_config.txt](https://github.com/fredlcore/BSB-LAN/files/10368208/BSB_LAN_config.txt)

Is that all right?

**Comment by 1coderookie** [comment]
You set device family and variant manually - did you try it with the default setting 0 / 0 for auto detection?

**Comment by marcelpettigrew** [comment]
Yes. 

I started with 0/0 and that was an attempt on my part.

**Comment by fredlcore** [comment]
Seems all fine so far for me, but really can't explain the behavior, so my only suggestion would be to completely install everything from scratch...

**Comment by marcelpettigrew** [comment]
I now had time to reinstall the system. It took much longer to create the file and there seems to be more data in it. 
[BSB-LAN-108-100-103941.txt](https://github.com/fredlcore/BSB-LAN/files/10466465/BSB-LAN-108-100-103941.txt)

**Comment by fredlcore** [comment]
Yes, now it's working. Can you also tell us what the reason was for this failure?

**Reply by marcelpettigrew** [reply, marked as answer]
I don't know exactly. I only reflashed the ESP. I did not change the wiring. Thanks for the help

**Comment by fredlcore** [comment]
Strange, but if that was a solution, I'll recommend that to the few users who have the same problem...
